trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './decorator'
+
+class Trimmerdecorator < Decorator
+  def correct_name
+    if @nameable.correct_name.length <= 10
+      @nameable.to_s
+    else
+      @nameable.correct_name[0..9].to_s
+    end
+  end
```

Comment:
- [ ] Your code fails when the length of the `@nameable.correct_name` is less or equal than 10, remember that `@nameable` in this case refers to an instance of a `Nameable`, in this order of ideas, returning the string version of it will end up in something like this: `"#<Nameable:0x00007f8d83669488>"`. Please consider modifying this logic to meet the expected result! 🤓 🔧 

💡 _Tip: you could do this in one line:_

```suggestion
  def correct_name
    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
  end




```